extensions/amp-story/amp-story.md

```diff
@@ -14,4 +14,373 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-# <a name="`amp-story`"></a> `amp-story`
\ No newline at end of file
+# <a name="`amp-story`"></a> `amp-story`
+
+**THIS IS EXPERIMENTAL AND UNDER ACTIVE DEVELOPMENT!!**
```

Comment:
Most AMP extensions start with a `<table>` here containing a description, required script, status, and links to examples, etc. See others for example.

Reply:
Filed #11530